### DISCUSSION TITLE
User Online Indicator in App

### DISCUSSION THREAD

**Discussion by benhaynes** [directus/directus#11868] [Feature Requests]
When a user is online, I'd like to see a green dot on their avatar/name throughout the App. Not sure how we pull this data, but it should be based on their activity... even when not changing pages. Ideally, we differentiate between online and idle (but logged in), with an option to explicitly set your status (menu from Current User Button).

**Comment by gensmann** [comment]
Would be nice as a general API point as well.

Maybe based on a global session timeout setting, measured by x seconds since last request?

**Reply by rijkvanzanten** [reply]
> Would be nice as a general API point as well.

That's the beauty of the app. Everything it does is using the regular API endpoints. By adding a feature like this to the app, we'd automatically add it as an API endpoint as well 😄

**Comment by rijkvanzanten** [comment]
Heya! Thank you for taking the time to submit this request! 

It has been over **90 days**, and this discussion has not received at least **15 votes** from the community. This means that we don't feel like there's enough community interest to warrant further R&D into this topic at this time. :ice_cube:

This request will now be closed to keep our discussions tidy. Please reach out if you have any questions!

_For more information, see our [Feature Request Process](https://docs.directus.io/contributing/feature-request-process.html)._
